values.yaml

```diff
@@ -206,6 +206,34 @@ matrix:
   # Human readable reason for why the homeserver is blocked
   disabledMessage: ""
 
+  # Matrix user verification service
+  uvs:
+    enabled: false
+    image:
+      repository: "matrixdotorg/matrix-user-verification-service"
+      tag: "v3.0.0"
+      pullPolicy: IfNotPresent
+    replicaCount: 1
+    # TODO: Find way how to generate access token without user interaction.
+    # Provide here access token for Matrix Synapse API.
+    # This must be token from admin user, currently can be fetched manually from Riot/Element dashboard.
+    accessToken: ""
```

Comment:
Access token is a secret. I think it should be instead a secret name and key here